src/components/app/CharacterDetail.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useParams, Link } from 'react-router-dom';
+import { useCharacterDetail } from '../../hooks.js';
+
+const DetailContainer = styled.div`
```

Comment:
Great idea, alternative you can do inline styling. Both are fine but if you want a different way, try looking into that.